storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -4938,13 +4931,13 @@ static bool xtrabackup_init_temp_log(void) {
     goto skip_modify;
   }
 
-  if (log_format < Log_format::VERSION_8_0_1) {
+  if (log_format < Log_format::CURRENT) {
```

Comment:
Upstream didn't bump the redo log header version right? :thinking: So the redo log version is same across 8.0.34 and 8.1?

Reply:
Done